modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -356,6 +356,80 @@
         end
       end
 
+      describe 'validation of claimant change of address elements' do
+        context 'when the country is valid' do
+          let(:country) { 'USA' }
+
+          it 'responds with a 200' do
+            with_okta_user(scopes) do |auth_header|
+              VCR.use_cassette('evss/claims/claims') do
+                VCR.use_cassette('brd/countries') do
+                  json = JSON.parse(data)
+                  json['data']['attributes']['changeOfAddress']['country'] = country
+                  data = json.to_json
+                  post path, params: data, headers: headers.merge(auth_header)
+                  expect(response).to have_http_status(:ok)
+                end
+              end
+            end
+          end
+        end
+
+        context 'when the country is invalid' do
+          let(:country) { 'United States of Nada' }
+
+          it 'responds with bad request' do
+            with_okta_user(scopes) do |auth_header|
+              VCR.use_cassette('evss/claims/claims') do
+                VCR.use_cassette('brd/countries') do
+                  json = JSON.parse(data)
+                  json['data']['attributes']['changeOfAddress']['country'] = country
+                  data = json.to_json
+                  post path, params: data, headers: headers.merge(auth_header)
+                  expect(response).to have_http_status(:bad_request)
+                end
+              end
+            end
+          end
+        end
+
+        context 'when no mailing address data is found' do
+          it 'responds with bad request' do
```

Comment:
Should this be updated to 'responds with a 422'?

Reply:
let's start leaving notes on these once we've made a decision. Or simply resolving the conversation if we went with the suggestions / comments. Assuming since this PR had previous approval, we're good to approve it, but be good to get in that habit. 👍